TestAssets/TestProjects/DesktopUsingPackageWithSatellites/DesktopUsingPackageWithSatellites.csproj

```diff
@@ -0,0 +1,12 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net46</TargetFramework>
```

Comment:
In #1132, which was closed as a dupe of #1006 which this PR purports to fix, the repro occurs only when `TargetFrameworks` is defined in the project and `TargetFramework` is now. Are we sure it fixes #1132? Should we add another test to covers that scenario?